packages/react-components/react-nav-preview/src/components/NavCategoryItem/useNavCategoryItem.styles.ts

```diff
@@ -1,57 +1,78 @@
-import { makeResetStyles, mergeClasses, makeStyles } from '@griffel/react';
-import { typographyStyles } from '@fluentui/react-theme';
+import { makeStyles, mergeClasses } from '@griffel/react';
 import { SlotClassNames } from '@fluentui/react-utilities';
+import { typographyStyles } from '@fluentui/react-theme';
 
 import type { NavCategoryItemSlots, NavCategoryItemState } from './NavCategoryItem.types';
+import {
+  useContentStyles,
+  useIconStyles,
+  useIndicatorStyles,
+  useRootDefaultClassName,
+} from '../sharedNavStyles.styles';
 
 export const navCategoryItemClassNames: SlotClassNames<NavCategoryItemSlots> = {
   root: 'fui-NavCategoryItem',
+  icon: 'fui-NavCategoryItem__icon',
+  content: 'fui-NavCategoryItem__content',
   expandIcon: 'fui-NavCategoryItem__expandIcon',
 };
 
-/**
- * Styles for the root slot
- */
-const useRootStyles = makeResetStyles({
-  display: 'flex',
-  ...typographyStyles.body1,
-});
-
-const useContentStyles = makeStyles({
-  icon: {
-    display: 'flex',
+const useExpandIconStyles = makeStyles({
+  base: {
+    marginInlineStart: 'auto',
   },
   open: {
-    transform: 'rotate(-90deg)',
-  },
-  closed: {
     transform: 'rotate(90deg)',
   },
   selected: typographyStyles.body1Strong,
 });
 
+/**
+ * Styles for the root slot
+ */
+export const useUniqueNavCategoryRootDefaultStyles = makeStyles({
+  base: {
+    width: '100%',
+  },
+});
+
```

Comment:
nit, I think this can keep a simple name. The other one is a classname, not a styles object so it doesn't clash.
```suggestion
export const useRootStyles = makeStyles({
  base: {
    width: '100%',
  },
});
```

Reply:
updated :-)